tests/references/signals.exponential_sweep.py

```diff
@@ -0,0 +1,7 @@
+# Write exponential sweep to csv for testing.
+# The sweep was manually inspected.
```

Comment:
Could you give a little explanation on the inspected criteria? This would help developers with less experience for similar implementations in the future.